src/components/TodoList/index.ts

```diff
@@ -0,0 +1 @@
+export * from './TodoList';
```

Comment:
Consider using a clear way to import and export by importing and then exporting to maintain consistency.